cdap-app-fabric/src/main/java/io/cdap/cdap/app/guice/AuthorizationModule.java

```diff
@@ -53,14 +55,13 @@
  * {@link PrivateModule} for authorization classes. This module is necessary and must be in
  * app-fabric because classes like {@link DatasetFramework}, {@link DynamicDatasetCache}, {@link
  * DefaultAdmin} are not available in cdap-security.
- *
  * This module is part of the injector created in StandaloneMain and MasterServiceMain, which makes
  * it available to services. The requirements for this module are: 1. This module is used for
  * creating and exposing {@link AccessControllerInstantiator}. 2. The {@link
  * AccessControllerInstantiator} needs a {@link DefaultAuthorizationContext}. 3. The {@link
  * DefaultAuthorizationContext} needs a {@link DatasetContext}, a {@link Admin} and a {@link
  * Transactional}.
- *
+ * -
```

Comment:
Is this intended?

Reply:
yeah I think having a space / empty line helps in better readability. 
So I put a `-` to avoid checkstyle issues

Reply:
Note for future: I think you can use `<p>` with empty lines. This would also make it better in rendered JavaDoc